### DISCUSSION TITLE
Can I clone this website's style and layout.

### DISCUSSION THREAD

**Discussion by hariienesh** [NvChad/nvchad.github.io#152] [Q&A] [answered]
Hi,

I want to clone this website style and layout for my personal use.

I will not use any of the content of this site.

If you allow so, please help me to clone this and deploy. I am having a real hard time trying to deploy to GitHub Pages and running it locally gives me a 404 error.

Thanks,

**Comment by siduck** [comment, marked as answer]
Hi @CosmicCodeBase , sure!   btw edit this file https://github.com/NvChad/nvchad.github.io/blob/src/.github/workflows/deploy.yml 

update "src" to your branch name

**Reply by hariienesh** [reply]
Thank you for giving me permission. I am using the 'src' branch, but when I try to deploy it, the README.md file from the GitHub repository is being deployed instead. When I run `npm start` locally, I get a 404 error, but it works if I use the `npm run dev` command. Can you tell me how to run the 'Deploy' workflow?
